src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordPropertySymbol.cs

```diff
@@ -16,15 +16,13 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     internal sealed class SynthesizedRecordPropertySymbol : SourceOrRecordPropertySymbol
```

Comment:
>SynthesizedRecordPropertySymbol [](start = 26, length = 31)

It looks like changes made in this file are going to conflict with [https://github.com/dotnet/roslyn/pull/44883](https://github.com/dotnet/roslyn/pull/44883)